ipld/retriever.go

```diff
@@ -56,13 +58,55 @@ func (r *Retriever) Retrieve(ctx context.Context, dah *da.DataAvailabilityHeader
 			if err == nil {
 				return eds, nil
 			}
+			if byzErr := r.handleByzantineError(ctx, dah, err); byzErr != nil {
+				return nil, byzErr
+			}
 		}
 		// retry quadrants until we can reconstruct the EDS or error out
 	}
 
 	return nil, format.ErrNotFound
 }
 
+func (r *Retriever) handleByzantineError(
+	ctx context.Context,
+	dah *da.DataAvailabilityHeader,
+	byzErr error,
+) error {
+	var errRow *rsmt2d.ErrByzantineRow
+	var errCol *rsmt2d.ErrByzantineCol
```

Comment:
```suggestion
        var (
           errRow *rsmt2d.ErrByzantineRow
           errCol *rsmt2d.ErrByzantineCol
        )
```

pls lint after applying suggestion